### PR TITLE
Added courses module

### PR DESCRIPTION
**Pull request by goar5670**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .idea/*
 .env
+atc-env
```
Comment (AKurmazov) [inline .gitignore]: Remove this, please

File: atc/wsgi.py [hunk 2]
```diff
@@ -8,9 +8,12 @@
 """
 
 import os
-
+# from dotenv import load_dotenv
 from django.core.wsgi import get_wsgi_application
 
+# project_folder = os.path.expanduser('C:\Users\BL7Awy\Documents\GitHub\atc')
+# load_dotenv(os.path.join(project_folder, '.env'))
```
Comment (AKurmazov) [inline atc/wsgi.py]: All the changes above are redundant

File: atc/settings.py [hunk 3]
```diff
@@ -9,15 +9,16 @@
...
 DEBUG = bool(int(os.environ.get('DEBUG', 1)))
 
-ALLOWED_HOSTS = ['0.0.0.0', ]
+ALLOWED_HOSTS = ['0.0.0.0', 'localhost']
 
 
 # Application definition
 
 INSTALLED_APPS = [
+    'courses.apps.CoursesConfig',
```
Comment (AKurmazov) [inline atc/settings.py]: Why you add `courses.app.CoursesConfig`, but not just `courses`

File: courses/models.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    ('4', '14:20-15:50'),
+    ('5', '16:00-17:30'),
+    ('6', '17:40-19:10'),
+    ('7', '19:20-20:50')
+]
+
+
+class Course(models.Model):
+    code = models.CharField(max_length=16, blank=True)
+    name = models.CharField(max_length=128, blank=True)
```
Comment (AKurmazov) [inline courses/models.py]: Why `blank=True`?

Comment (AKurmazov) [reply courses/models.py]: Both for `code` and `name`

File: courses/models.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    
+    def __str__(self):
+        return f'{self.name}'
+
+
+class Class(models.Model):
+    day = models.CharField(max_length=9, choices=DAY_CHOICES)
+    _type = models.CharField(max_length=8, choices=TYPE_CHOICES)
+    timeslot = models.CharField(max_length=11, choices=TIMESLOT_CHOICES)
+    course = models.ForeignKey(Course, on_delete=models.CASCADE)
```
Comment (AKurmazov) [inline courses/models.py]: Add parameter `related_name='classes'`

File: courses/models.py [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+    def __str__(self):
+        return f'{self.course} - {self.get__type_display()}'
+
+
+class Room(models.Model):
+    number = models.IntegerField(default=0)
```
Comment (AKurmazov) [inline courses/models.py]: Why `default=0`?

File: docker-compose.yml [hunk 7]
```diff
@@ -11,8 +11,8 @@ services:
     volumes:
       - .:/code
     command: >
-      bash -c "python manage.py migrate &&
-      python manage.py runserver 0.0.0.0:8000"
+      python manage.py runserver 0.0.0.0:8000 
+
```
Comment (AKurmazov) [inline docker-compose.yml]: Remove the empty line

File: .gitignore (hunk 1 above)
Comment (goar5670) [reply .gitignore]: done

**Review by AKurmazov — APPROVED** [review]

**Pull request merged**